web/jobs/jobs.go

```diff
@@ -23,11 +23,20 @@ type (
 		Arguments json.RawMessage  `json:"arguments"`
 		Options   *jobs.JobOptions `json:"options"`
 	}
-
 	apiQueue struct {
 		Count      int `json:"count"`
 		workerType string
 	}
+	apiTrigger struct {
```

Comment:
Indeed I could. Though the `triggerDoc` does only implement the `couchdb.Doc` interface, so I would have to add some methods.

However I don't want to fix that in this PR since I would like to introduce a more general approach to fix this issue: by having a simple struct in the `jsonapi` that would implement the `jsonapi.Object` and that we could create on the fly for these type of objects that do not implement the `jsonapi.Object` interface.